recipes/cfg-cli/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cfg-cli" %}
+{% set version = "0.1.32" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0786048d63e5b66338307fb9739142aa06a090744e00141c95c0b8054b4c767b
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - prism = cfg_cli.prism_cli:wrapper
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - poetry
+  run:
+    - boto3 >=1.14.11,<2.0.0
+    - fire >=0.2.1,<0.3.0
+    - importlib-metadata >=1.7.0,<2.0.0
+    - prompt-toolkit >=3.0.5,<4.0.0
+    - python >=3.6
+    - python-jose >=3.1.0,<4.0.0
+    - requests >=2.23.0,<3.0.0
+    - tabulate >=0.8.7,<0.9.0
+
+test:
+  imports:
+    - cfg_cli
+    - cfg_cli.clients
+    - cfg_cli.errors
+  commands:
+    - prism --help
+
+about:
+  home: "https://github.com/CofactorGenomics/immunoprism-cli"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "Command-line tool for Cofactor Genomics' products and services."
+  doc_url: "https://cofactorgenomics.github.io"
+
+extra:
+  recipe-maintainers:
+    - alexbode
+    - davemessina
```

Comment:
@davemessina are you OK with being a maintainer?

Reply:
@scopatz yep, thank you!